code/game/machinery/Sleeper.dm

```diff
@@ -245,29 +245,24 @@
 		if(M.Victim == L)
 			usr << "[L.name] will not fit into the sleeper because they have a slime latched onto their head."
 			return
+
 	if(L == user)
-		visible_message("[user] starts climbing into \the [src].", 3)
+		visible_message("[user] climbs into \the [src].", 3)
 	else
-		visible_message("[user] starts putting [L.name] into \the [src].", 3)
+		visible_message("[user] places [L.name] into \the [src].", 3)
 
-	if(do_after(user, src, 20))
-		if(src.occupant)
-			user << "<span class='notice'><B>The sleeper is already occupied!</B></span>"
-			return
-		if(!L || L.buckled) return
-		L.loc = src
-		L.reset_view()
-		src.occupant = L
-		update_icon()
-		L << "<span class='notice'><b>You feel cool air surround you. You go numb as your senses turn inward.</b></span>"
-		for(var/obj/OO in src)
-			OO.loc = src.loc
-		src.add_fingerprint(user)
-		if(user.pulling == L)
-			user.stop_pulling()
-		if(!(stat & (BROKEN|NOPOWER)))
-			set_light(light_range_on, light_power_on)
-		return
+	L.loc = src

```

Comment:
You should still be using forceMove here regardless, it's how lighting overlays and various other things update.